charon/src/ast/llbc_ast.rs

```diff
@@ -59,6 +59,7 @@ pub enum RawStatement {
     Sequence(Box<Statement>, Box<Statement>),
     Switch(Switch),
     Loop(Box<Statement>),
+    Error,
```

Comment:
I suggest adding a string with the unsupported statement: it can be useful for debugging (Lucas does this a lot in hax for instance).